### ISSUE TITLE
Missed Recently Added Notifications

### ISSUE THREAD

**Issue by Kedryn** [Tautulli/Tautulli#1234]
When plex finds new media, all of them appear in the recently added section on the PlexPy home page. But only few of them are notified to the agents. I have both email and telegram agents active. They receive same shows, they miss same shows notifications.

There is nothing in logs about missed ones.

**Comment by JonnyWong16** [comment]
It's not perfect. Sometimes stuff gets missed. If you are using the v2-beta then you can go manually trigger a recently added notification.

And you didn't read the [issues guidelines](https://github.com/JonnyWong16/plexpy#issues) before posting.

**Comment by Kedryn** [comment]
I'm using 1.4.25, no new version is found if i do a check update.
I did restart serveral times, same result.
Logs just shows some:


2018-01-31 00:07:25 | DEBUG | PlexPy Monitor :: Library item 112677 has been added to Plex.
-- | -- | --
2018-01-31 00:07:25 | WARNING | PlexPy Pmsconnect :: Unable to parse XML for get_metadata: 'NoneType' object has no attribute 'getElementsByTagName'.
2018-01-31 00:07:25 | WARNING | Failed to access uri endpoint /library/metadata/116400 with error ('The read operation timed out',)

But i don't know if they are for the missed show notification. There is no other errors in logs. PlexPy and Plex are on the same machine, plenty of cpu and ram, no slow downs.
